frontend/src/components/VaultDetails.vue

```diff
@@ -108,90 +108,101 @@
           </li>
         </ul>
       </div>
+    </div>
 
-      <!-- button bar -->
-      <div>
-        <h3 class="font-medium text-gray-900">{{ t('vaultDetails.actions.title') }}</h3>
-
-        <p v-if="paymentRequired" class="text-sm text-red-900 mt-1">
-          {{ t('vaultDetails.error.paymentRequired') }}
-        </p>
-
-        <div v-else-if="!vault.archived" class="mt-2 flex flex-col gap-2">
-          <div class="flex gap-2">
-            <button :disabled="usersRequiringAccessGrant.length == 0" type="button" class="flex-1 bg-primary py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-white hover:bg-primary-d1 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary disabled:opacity-50 disabled:hover:bg-primary disabled:cursor-not-allowed" @click="showGrantPermissionDialog()">
-              {{ t('vaultDetails.actions.updatePermissions') }}
-            </button>
-            <button type="button" class="bg-white py-2 px-4 border border-gray-300 rounded-md shadow-sm text-sm font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary" @click="reloadDevicesRequiringAccessGrant()">
-              <span class="sr-only">{{ t('vaultDetails.actions.updatePermissions.reload') }}</span>
-              <ArrowPathIcon class="h-5 w-5" aria-hidden="true" />
-            </button>
-          </div>
-          <button type="button" class="bg-white py-2 px-4 border border-gray-300 rounded-md shadow-sm text-sm font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary" @click="showEditVaultMetadataDialog()">
-            {{ t('vaultDetails.actions.editVaultMetadata') }}
-          </button>
-          <button type="button" class="bg-white py-2 px-4 border border-gray-300 rounded-md shadow-sm text-sm font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary" @click="showDownloadVaultTemplateDialog()">
-            {{ t('vaultDetails.actions.downloadVaultTemplate') }}
-          </button>
-          <button type="button" class="bg-white py-2 px-4 border border-gray-300 rounded-md shadow-sm text-sm font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary" @click="showRecoveryKeyDialog()">
-            {{ t('vaultDetails.actions.showRecoveryKey') }}
-          </button>
-          <button type="button" class="bg-red-600 py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-white  hover:bg-red-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500" @click="showArchiveVaultDialog()">
-            {{ t('vaultDetails.actions.archiveVault') }}
-          </button>
-        </div>
+    <!-- button bar -->
+    <div >
```

Comment:
Not sure if this could be added as a linter rule, but `<div >` has a space. :smile:

Reply:
Fixed in  79d980d37177daab6df0ff4cc75b24a85da2d41b